src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -31,7 +39,24 @@
 @WebServlet("/events/*")
 public class EventServlet extends HttpServlet {
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String pathName = request.getPathInfo();
+    UserService userService = UserServiceFactory.getUserService();
+    if (pathName.length() <= 1) {
```

Comment:
There might be a better way to check for bad requests.

Reply:
Deleted it